base/task/current_thread.h

```diff
@@ -275,15 +275,14 @@ class BASE_EXPORT CurrentIOThread : public CurrentThread {
   typedef base::MessagePumpIOStarboard::SocketWatcher SocketWatcher;
   typedef base::MessagePumpIOStarboard::IOObserver IOObserver;
 
-  enum Mode{WATCH_READ = base::MessagePumpIOStarboard::WATCH_READ,
-            WATCH_WRITE = base::MessagePumpIOStarboard::WATCH_WRITE,
-            WATCH_READ_WRITE = base::MessagePumpIOStarboard::WATCH_READ_WRITE};
-
   bool Watch(SbSocket socket,
```

Comment:
Should this be renamed `RegisterInterests`? And should `UnregisterInterest` be pluralized as it looks like it can unregister multiple?

Reply:
The `UnregisterInterest` was named after the similar functioning method in upstream `MessagePumpEpoll`, but actually isn't needed here in CurrentThread. Removed.